src/scss/components/optional/_page-banner.scss

```diff
@@ -1,69 +1,140 @@
+$spaceForColouredRays: 20px;
+$spaceForColouredRays-tiny: 60px;
+$spaceForColouredRays-mid: 164px;
+$spaceForColouredRays-wide: 148px;
+
 
 @mixin pageBanner() {
 
-    .c-pageBanner {
-        background-color: $white;
-        margin-bottom: -#{spacing(x2)};
-        max-height: 300px;
-        min-height: 100px;
-        overflow: hidden;
-        position: relative;
+   .c-pageBanner {
+       background-color: $white;
+       max-height: 300px;
+       min-height: 100px;
+       overflow: hidden;
+       position: relative;
 
-        @include media('>=wide') {
-            max-height: 400px;
-        }
+       @include media('>=wide') {
+           max-height: 400px;
+       }
+   }
 
-        // Prevent IE from applying these styles as it can't correctly position the rays
-        @supports (position: absolute) {
-            &:after {
-                background-image: inline('#{$img-path}/decoration/rays--white--small.svg');
-                background-position: bottom left;
-                background-repeat: no-repeat;
-                bottom: -1px;
-                content: '';
-                height: 100%;
-                left: 0;
-                position: absolute;
-                width: 100%;
-                z-index: zIndex(low);
-
-                @include media('>=mid') {
-                    background-image: inline('#{$img-path}/decoration/rays--white.svg');
-                }
-            }
-        }
-    }
+   .c-pageBanner--wide {
+       max-height: 600px;
 
-        // any elements below the banner will be moved up, over the bottom part of the banner
-        .c-pageBanner--negativeBottom--aboveMid {
-            @include media('>=mid') {
-                margin-bottom: -115px;
-            }
+       &.c-pageBanner--withRays--coloured {
+           .c-pageBanner-img {
+               margin-bottom: $spaceForColouredRays;
 
-            @include media('>=wide') {
-                margin-bottom: -150px;
-            }
-        }
+               @include media('>=tiny') {
+                   margin-bottom: $spaceForColouredRays-tiny;
+               }
+
+               @include media('>=mid') {
+                   margin-bottom: $spaceForColouredRays-mid;
+               }
+
+               @include media('>=wide') {
+                   margin-bottom: $spaceForColouredRays-wide;
+               }
+           }
+       }
+   }
+
+   .c-pageBanner--narrow {
+       max-height: 300px;
+
+       @include media('>=huge') {
+           max-height: 370px;
+       }
+       .c-pageBanner-img {
+           @include media('>=mid') {
+               margin-top: -8%;
+           }
+       }
+   }
+
+   .c-pageBanner--withRays--white {
+       // Prevent IE from applying these styles as it can't correctly position the rays
+       @supports (position: absolute) {
+           &:after {
+               background: inline('#{$img-path}/decoration/rays--white--narrow.svg') no-repeat bottom left;
```

Comment:
Worth checking the file size difference in inlining all these SVGs.  May be better to just include as URL includes if it's a decent file size difference?

Reply:
✅